packages/wdio-sauce-service/sauce-service.d.ts

```diff
@@ -1,151 +0,0 @@
-declare module WebdriverIO {
```

Comment:
we can't remove this file as the package.json points to it https://github.com/webdriverio/webdriverio/pull/6001/files#diff-1e974f46805437badbcd51209a7a5b716c7c08ec4b9c1554ffcbc9ca9fa48ef3R45

Reply:
I didn't remove it though - only its contents, and I did so since they are outdated (and now included in `node-saucelabs`, do we need to do anything differently here?

Reply:
let's keep this file for now because it is still being referred to in the package.json (`"types": "sauce-service.d.ts"`)